src/tests/ftest/rebuild/container_create.py

```diff
@@ -0,0 +1,258 @@
+
+        # Get ior params
+        ior_cmd = IorCommand()
+        ior_cmd.get_params(self)
+
+        # Cancel any tests with tickets already assigned
+        if rank == 1 or rank == 2:
+            self.cancelForTicket("DAOS-2434")
+
+        errors = [0 for _ in range(loop_qty)]
+        for loop in range(loop_qty):
+            # Log the start of the loop
+            loop_id = "LOOP {}/{}".format(loop + 1, loop_qty)
+            self.log.info("%s", "-" * 80)
+            self.log.info("%s: Starting loop", loop_id)
+
+            # Create the requested number of pools
+            info_checks = []
+            rebuild_checks = []
+            for pool in self.pool:
+                pool.create()
+                info_checks.append(
+                    {
+                        "pi_uuid": pool.uuid,
+                        "pi_ntargets": node_qty * targets,
+                        "pi_nnodes": node_qty,
+                        "pi_ndisabled": 0,
+                    }
+                )
+                rebuild_checks.append(
+                    {
+                        "rs_errno": 0,
+                        "rs_done": 1,
+                        "rs_obj_nr": 0,
+                        "rs_rec_nr": 0,
+                    }
+                )
+
+            # Check the pool info
+            status = True
+            for index, pool in enumerate(self.pool):
+                status &= pool.check_pool_info(**info_checks[index])
+                status &= pool.check_rebuild_status(**rebuild_checks[index])
+            self.assertTrue(
+                status,
+                "Error verifying pool info prior to excluding rank {}".format(
+                    rank))
+
+            # Create a container with 1GB of data in the first pool
+            ior_cmd.flags.update("-v -w -W -G 1 -k", "ior.flags")
+            ior_cmd.daos_destroy.update(False, "ior.daos_destroy")
+            ior_cmd.set_daos_params(self.server_group, self.pool[0])
+            self.log.info(
+                "%s: Running IOR on pool %s to fill container %s with data",
+                loop_id, self.pool[0].uuid, ior_cmd.daos_cont.value)
+            self.run_ior(loop_id, ior_cmd)
+
+            # Exclude the first rank from the first pool to initiate rebuild
+            self.pool[0].start_rebuild(self.server_group, rank, self.d_log)
+
+            # Wait for rebuild to start
+            self.pool[0].wait_for_rebuild(True, 1)
+
+            # Create additional containers in the last pool
+            new_containers = self.add_containers_during_rebuild(
+                loop_id, cont_qty, self.pool[0], self.pool[-1])
+
+            # Confirm rebuild completes
+            self.pool[0].wait_for_rebuild(False, 1)
+
+            # Check the pool info
+            info_checks[0]["pi_ndisabled"] += targets
+            rebuild_checks[0]["rs_done"] = 1
+            rebuild_checks[0]["rs_obj_nr"] = ">=0"
+            rebuild_checks[0]["rs_rec_nr"] = ">=0"
```

Comment:
just curious, why not check rs_status?

Reply:
We do.  Because we still expect it to be 0, there is no need to update the expected value.

Reply:
Is that new?  I don't see it in daos_cref.py

Reply:
oh, sorry I mean rs_errno.

Reply:
`rs_errno` is still being verified.  The expected value is set to `0` back on lines 181-187.  Here we are just updating any values that should be different after the initial pool info verification.  In the case of `rs_errno` it should still be 0, so no update is required